### PR TITLE
Clarify example projects resource

### PR DESCRIPTION
**Pull request by Cyb3r-Jak3**
Make examples pages project resources clearer for  #2517

cc @WalshyDev

### CODE REVIEW

File: docs/resources/pages_project.md [hunk 1]
```diff
@@ -16,16 +16,16 @@ connected GitHub or GitLab account connected to Cloudflare. See the
 ## Example Usage
 
 ```terraform
-# Basic project
+# A direct upload Pages project
 resource "cloudflare_pages_project" "basic_project" {
-  account_id        = "f037e56e89293a057740de681ac9abbe"
+  account_id        = var.account_id
```
Comment (jacobbednarz) [inline docs/resources/pages_project.md:21]: let's leave these references to a static account ID.

File: internal/sdkv2provider/schema_cloudflare_pages_project.go [hunk 2]
```diff
@@ -240,14 +240,14 @@ func resourceCloudflarePagesProjectSchema() map[string]*schema.Schema {
 			Required:    true,
 		},
 		"build_config": {
-			Description: "Configuration for the project build process.",
+			Description: "Configuration for the project build process. Read more about the build configuration here: https://developers.cloudflare.com/pages/platform/build-configuration",
```
Comment (jacobbednarz) [inline internal/sdkv2provider/schema_cloudflare_pages_project.go:243]:
```suggestion
			Description: "Configuration for the project build process. Read more about the build configuration in the [developer documentation](https://developers.cloudflare.com/pages/platform/build-configuration)",
```

File: internal/sdkv2provider/schema_cloudflare_pages_project.go [hunk 3]
```diff
@@ -240,14 +240,14 @@ func resourceCloudflarePagesProjectSchema() map[string]*schema.Schema {
...
 			Type:        schema.TypeList,
 			Elem:        &buildConfig,
 			MaxItems:    1,
 			Optional:    true,
 		},
 		"source": {
-			Description: "Configuration for the project source.",
+			Description: "Configuration for the project source. Read more about the source configuration here: https://developers.cloudflare.com/pages/platform/branch-build-controls/",
```
Comment (jacobbednarz) [inline internal/sdkv2provider/schema_cloudflare_pages_project.go:250]:
```suggestion
			Description: "Configuration for the project source. Read more about the source configuration in the [developer documentation](https://developers.cloudflare.com/pages/platform/branch-build-controls/)",
```

File: examples/resources/cloudflare_pages_project/resource.tf [hunk 4]
```diff
@@ -98,3 +98,89 @@ resource "cloudflare_pages_project" "deployment_configs" {
...
+      compatibility_date  = "2022-08-15"
+      compatibility_flags = ["preview_flag"]
+    }
+    production {
+      environment_variables = {
+        ENVIRONMENT = "production"
+        OTHER_VALUE = "other value"
+      }
+      secrets = {
+        TURNSTILE_SECRET = var.turnstile_secret
```
Comment (jacobbednarz) [inline examples/resources/cloudflare_pages_project/resource.tf:161]: let's use fake but usable values here. assuming people are using variables makes this a little harder to grok for new comers.

File: docs/resources/pages_project.md [hunk 5]
```diff
@@ -176,27 +176,28 @@ resource "cloudflare_pages_project" "deployment_configs" {
         OTHER_VALUE = "other value"
       }
       secrets = {
-        TURNSTILE_SECRET = var.turnstile_secret
-        TURNSTILE_INVIS_SECRET = var.turnstile_invisible_secret
+        TURNSTILE_SECRET       = "1x0000000000000000000000000000000AA"
+        TURNSTILE_INVIS_SECRET = "2x0000000000000000000000000000000AA"
+
+        kv_namespaces = {
```
Comment (WalshyDev) [inline docs/resources/pages_project.md]: This is now all in the secrets block

File: docs/resources/pages_project.md [hunk 6]
```diff
@@ -176,27 +176,28 @@ resource "cloudflare_pages_project" "deployment_configs" {
...
+        r2_buckets = {
+          R2_BINDING_1 = "some-bucket"
+          R2_BINDING_2 = "other-bucket"
+        }
+        d1_databases = {
+          D1_BINDING_1 = "445e2955-951a-4358-a35b-a4d0c813f63"
+          D1_BINDING_2 = "a399414b-c697-409a-a688-377db6433cd9"
+        }
+        compatibility_date  = "2022-08-16"
+        compatibility_flags = ["production_flag", "second flag"]
```
Comment (WalshyDev) [inline docs/resources/pages_project.md:199]: we should probably use some real flags haha, what about `nodejs_compat` and `streams_enable_constructors`

File: examples/resources/cloudflare_pages_project/resource.tf [hunk 7]
```diff
@@ -158,27 +158,28 @@ resource "cloudflare_pages_project" "deployment_configs" {
...
-        R2_BINDING_1 = "some-bucket"
-        R2_BINDING_2 = "other-bucket"
-      }
-      d1_databases = {
-        D1_BINDING_1 = "445e2955-951a-4358-a35b-a4d0c813f63"
-        D1_BINDING_2 = "a399414b-c697-409a-a688-377db6433cd9"
+        TURNSTILE_SECRET       = "1x0000000000000000000000000000000AA"
+        TURNSTILE_INVIS_SECRET = "2x0000000000000000000000000000000AA"
+
+        kv_namespaces = {
```
Comment (WalshyDev) [inline examples/resources/cloudflare_pages_project/resource.tf:164]: same here, incorrectly in secrets block

**Pull request merged**